CONTRIBUTING.rst

```diff
@@ -0,0 +1,120 @@
+============
+Contributing
+============
+
+Contributions are always welcome, and they are greatly appreciated! Every little
+bit helps, and credit will always be given. You can contribute in many ways:
+
+Types of Contributions
+----------------------
+
+Report Bugs
+~~~~~~~~~~~
+
+Report bugs at https://github.com/scrapinghub/spidermon/issues.
+
+If you are reporting a bug, please include:
```

Comment:
How about making an issue template and removing the list from here?

Reply:
I like the idea of issues templates, but I also want to have some guidance for new contributors as soon as possible. I use other scrapinghub projects as reference for this file, so what do you think creating this first version and then we discuss issue template in a different PR?